### PR TITLE
RD-3132 RD-3133 Add common backend configurations for tsc and eslint

### PR DESCRIPTION
**Pull request by kubama**
## Description
Added common backend configurations for tsc and eslint

## Screenshots / Videos
N/A

## Checklist

- [X] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [x] I followed the [UI Testing Guidelines](https://cloudifysource.atlassian.net/l/cp/udCVfvrx) and verified that all tests/checks have passed.
- [X] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [X] I added proper labels to this PR.

## Tests
N/A

 

## Documentation
Updated README

### CODE REVIEW

File: packages/configs/tsconfig.node.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "extends": "./tsconfig.base.json",
+    "compilerOptions": {
+        "lib": ["es2021"],
+        "module": "commonjs",
+        "target": "es2021"
+    },
+
```
Comment (qooban) [inline packages/configs/tsconfig.node.json:8]: If in `packages/configs/eslint-common-backend.json` we are referring to specific paths, maybe here we can also keep `"exclude": ["**/__mocks__/*", "test"]` (which is present in both - Stage and Composer - backend TS configs)?

File: packages/configs/README.md [hunk 2]
```diff
@@ -37,13 +37,14 @@ The table below describes their purpose and dependencies necessary to be install
...
 | `eslint-common-node.json`    | Node-based projects       | all from `eslint-config-airbnb-base` and `eslint-common.json`, `eslint-plugin-node`                                                                                                                |
 | `eslint-common-cypress.json` | Cypress subprojects       | all from `eslint-config-airbnb-base` and `eslint-common.json`, `eslint-plugin-cypress`, `eslint-plugin-chai-friendly`                                                                              |
 | `eslint-common-jest.json`    | Jest subprojects          | `eslint-plugin-jest`                                                                                                                                                                               |
+| `eslint-common-backend.json` | Backends                  | `eslint-common-node.json`, `eslint-common-jest.json`                                                                                                                                                                        |
 
 
 ### TypeScript
```
Comment (qooban) [inline packages/configs/README.md:50]: I guess we should enhance this section with `tsconfig.node.json` description, what do you think?

File: packages/configs/eslint-common-backend.json [hunk 3]
```diff
@@ -0,0 +1,50 @@
+{
+    "overrides": [
+        {
+            "files": "./backend/**/*.ts",
+            "extends": [
+                "./eslint-common-node.json",
+                "plugin:node/recommended-module"
+            ],
+            "rules": {
+                "prefer-promise-reject-errors": "off",
```
Comment (qooban) [inline packages/configs/eslint-common-backend.json:10]: Hm... How difficult would it be to remove this line? As we implemented RD-571 (I know it was about unhandled rejections and not necessarily about using error throwing), maybe not that big? 🤔

File: packages/configs/tsconfig.node.json (hunk 1 above)
Comment (kubama) [reply packages/configs/tsconfig.node.json:8]: I tried this but it didn't work - looks like the `exclude` property is not picked up from the extended configuration.

File: packages/configs/eslint-common-backend.json (hunk 3 above)
Comment (kubama) [reply packages/configs/eslint-common-backend.json]: Checked and it looks like we can just remove it without further alignments - done.

File: packages/configs/README.md (hunk 2 above)
Comment (kubama) [reply packages/configs/README.md:50]: Done

**Review by qooban — APPROVED** [review]
Looks good! 👍🏻

**Pull request merged**